pcsx2-qt/MainWindow.cpp

```diff
@@ -736,16 +782,31 @@ void MainWindow::onVideoCaptureToggled(bool checked)
 		return;
 	}
 
-	const QString container(QString::fromStdString(
-		Host::GetStringSettingValue("EmuCore/GS", "CaptureContainer", Pcsx2Config::GSOptions::DEFAULT_CAPTURE_CONTAINER)));
-	const QString filter(tr("%1 Files (*.%2)").arg(container.toUpper()).arg(container));
-
-	QString path(QStringLiteral("%1.%2").arg(QString::fromStdString(GSGetBaseVideoFilename())).arg(container));
-	path = QDir::toNativeSeparators(QFileDialog::getSaveFileName(this, tr("Video Capture"), path, filter));
-	if (path.isEmpty())
-		return;
+	if (s_record_on_start && !s_path_to_recording_for_record_on_start.isEmpty()) 
+	{	
+		QMessageBox msgbox2(this);
```

Comment:
I'm not a huge fan of using a messagebox for the purpose of notifications that don't require the user to do anything.

I would opt to use our OSD system instead. I experimented and found that the following works well.

```cpp
Host::AddOSDMessage(tr("Recording will start in a moment").toStdString(), 3.0f);
QTimer::singleShot(2000, []() { g_emu_thread->beginCapture(s_path_to_recording_for_record_on_start); });
```